server/src/main/java/org/spine3/server/entity/storage/EntityQuery.java

```diff
@@ -59,14 +59,15 @@
  * considered matching.
  *
  * @param <I> the type of the IDs of the query target
- *
  * @author Dmytro Dashenkov
  * @see EntityRecordWithColumns
  */
-public final class EntityQuery<I> {
+public final class EntityQuery<I> implements Serializable {
+
+    private static final long serialVersionUID = 7007336125384634462L;
```

Comment:
Let's use `1L` for now. It's more manageable if we really need to increment it to the next version someday.